main.go

```diff
@@ -41,6 +42,7 @@ func main() {
 	cmd.AddCommand(createcmd.NewCommand())
 	cmd.AddCommand(destroycmd.NewCommand())
 	cmd.AddCommand(dumpcmd.NewCommand())
+	cmd.AddCommand(consolelogs.NewCommand())
```

Comment:
Not sure if we care but this doesn't follow the pattern of the `infra` and `iam` command of `hypershift $inframanagementcommand $cloudprovider` as `aws` is not part of it